django_netjsonconfig/base/config.py

```diff
@@ -92,22 +92,12 @@ def _has_device(self):
         return hasattr(self, 'device')
 
     def get_context(self):
-        """
-        additional context passed to netjsonconfig
-        """
+        if not self._has_device():
+            return app_settings.CONTEXT
         c = {}
-        if self._has_device():
-            c.update({
-                'id': str(self.device.id),
-                'key': self.key,
-                'name': self.name,
-                'mac_address': self.mac_address
-            })
-            if self.context:
-                c.update(self.context)
-        c.update(app_settings.CONTEXT)
-        if app_settings.HARDWARE_ID_ENABLED and self._has_device():
-            c.update({'hardware_id': self.device.hardware_id})
+        if self.context:
+            c.update(self.context)
```

Comment:
ups, I didn't notice this, I forgot we had this feature, sorry for my bad advice, we need to rollback to how it was in the previous round of review, and we need to make sure the current config instance is being used, unless we're dealing with a new instance, in which case we should create a new one on the fly.

Reply:
@nemesisdesign so I have to revert changes made in /base/device.py and /base/config.py?